twoliter/embedded/build.Dockerfile

```diff
@@ -150,12 +150,10 @@ RUN --mount=target=/host \
 # Ensure that the target binutils that `find-debuginfo.sh` uses are present in $PATH.
 ENV PATH="/usr/${ARCH}-bottlerocket-linux-gnu/debuginfo/bin:${PATH}"
 
-# We use the "nocache" writable space to generate code where necessary, like the variant-
-# specific models.
+# We use the "nocache" writable space to generate code where necessary.
```

Comment:
You can also get rid of the `variantcache` stage above

Reply:
Done. Is the comment outdated as well? Just delete that comment?

Reply:
> Is the comment outdated as well? Just delete that comment?

You can delete it. I'm not sure what "nocache" refers to, probably a slightly older way we generated that mount.